### ISSUE TITLE
After upgrade to 2.0.3 Modified keybindings no longer work.

### ISSUE THREAD

**Issue by ConundrumAdInfinitum** [u8sand/Baka-MPlayer#157]
The new version of the program ignores the keybindings in the "bakamplayer.ini" file in the program's root directory on Windows x86_64.

**Comment by ConundrumAdInfinitum** [comment]
After Deleting the previous installation (folder containing Baka-MPlayer) and trying fresh with 2.0.3 on Windows, and re-setting the Preferences for the Key Bindings from scratch, I'm having mixed results where the keybindings will sometimes take, and sometimes not work.

**Comment by ConundrumAdInfinitum** [comment]
I think custom key bindings need to be fixed.

**Comment by u8sand** [comment]
@ConundrumAdInfinitum Is it the keybindings that don't work or is it the preferences dialog? I know of a preference dialog bug w/ keybindings but I was under the impression keybindings worked fine.

**Comment by ConundrumAdInfinitum** [comment]
Based on the way it is behaving it could be a bug in the preferences dialog panel. I did a clean install of Windows, and had to set up Baka-MPlayer again after reporting the problem. Fresh DL and install of 2.0.3 x86_64 on Windows 10. Preferences dialog acted a bit strangely: When you select to Change the keybinding, it created a duplicate preference at another place in the Preferences dialog, instead of just updating the keybinding. Additionally, the custom keybindings don't work UNLESS you DELETE the standard ones. I managed to get everything working how I like it, but I really hope I don't accidently "Reset All", ever. Hope this answers your question @u8sand
